### PR TITLE
docs for paradigm layouts

### PR DESCRIPTION
**Pull request by eddieantonio**

### CODE REVIEW

File: docs/paradigm-layouts-and-metadata.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Paradigm layouts
 ================
 
+> ⚠️ **NOTE** ⚠️ 
+>
+> This describes the legacy, NDS-style paradigm layout format for
+> itwêwina (crkeng). For documentation on the current format, please see
+> [this document instead](./paradigm-layouts.md).
+
```
Comment (andrewdotn) [inline docs/paradigm-layouts-and-metadata.md:9]: If the old format isn’t used at all anymore, can we delete this file?

**Review by andrewdotn — APPROVED** [review]

File: docs/paradigm-layouts.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+_productive_ (i.e., used for systematically for a large number of
+lexemes).
+
+A **static paradigm layout** is one in which there are no placeholders.
+All cells have their wordforms **explicitly** filled out. No
+substitution is needed at runtime, so their contents will be presented
+**as-is**.
+
+Static paradigm layouts are useful for
+
```
Comment (aarppe) [inline docs/paradigm-layouts.md]: Something missing here!

File: docs/paradigm-layouts.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+Cells are separated by one U+0009 CHARACTER TABULATION character
+(horizontal tab character).
+
+Each row in the TSV file _should_ contain the same number of tab
+characters. This is to facilitate display and editing of the TSV files
+in tools that expect an equal amount of tabs per row (e.g., the GitHub
+previewer, [VisiData][vd], common spreadsheet software,
+[PyCharm][pycharm-tsv]).
+
+Each panes is separated by one "blank" lines. A "blank" means any line
```
Comment (aarppe) [inline docs/paradigm-layouts.md]: Each *pane*

File: docs/paradigm-layouts.md (hunk 2 above)
Comment (eddieantonio) [reply docs/paradigm-layouts.md]: These comments are somehow associated with an old commit; the current commit does not contain either of the lines commented upon. 🤔

File: docs/paradigm-layouts.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+_dynamic wordform cells_. A paradigm layout is a _static layout_ when
+it contains zero _dynamic_ wordform cells. A _dynamic layout_ **MUST**
+be _filled_ before the layout is ready for presentation.
+
+Aside from the presence or absence of the `${lemma}` placeholder, the
+exact syntax of a wordform cell is defined by the implementation.
+
+### Partial Grammar
+
+Here is a partial W3C EBNF grammar of the layout file specification.
```
Comment (dwhieb) [inline docs/paradigm-layouts.md:128]: What's EBNF? Can you provide a link here?

File: docs/paradigm-layouts.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+ColumnLabel ::= ('|' SP Tag)+
+MissingForm ::= '-' '-'
+EmptyCell ::= SP*
+
+TAB ::= #x09
+NL  ::= #x0A
+SP  ::= #x20
+```
+
+### Example
```
Comment (dwhieb) [inline docs/paradigm-layouts.md:157]: This is super useful! Might even be worth putting this towards the top, followed by the explainer.

File: docs/paradigm-layouts.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+└── {paradigm-name}.tsv
+```
+
+Files directly within the `layouts` directory, can be:
+
+ * a `.tsv` layout file, whose filename stem (part before the `.tsv`) is
+   the _paradigm name_ that this layout corresponds to; or
+ * a subdirectory, whose filename is the _paradigm name_ this directory
+   corresponds to. Within this subdirectory are `.tsv` layout files that
+   indicate the different _size options_ available for this _paradigm
```
Comment (dwhieb) [inline docs/paradigm-layouts.md:203]: Might be good to briefly explain the fact that paradigms can come in different sizes towards the top too, maybe under **Components of the paradigm system**.

**Review by dwhieb — COMMENTED** [review]
The paradigm layouts doc is super helpful. I understand the system much better now.

Everything was clear and understandable. Just a couple minor suggestions.

**Pull request merged**